native/src/test/resources/packages/test-consolidate/modules/foo/foo.bal

```diff
@@ -0,0 +1,7 @@
+import ballerina/io;
+
+configurable string name = ?;
+
+public function foo() {
+    io:println("Hello, " + name + "!");
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in bd473123358220cca26188349581184775fe47ca